jinja2/utils.py

```diff
@@ -480,12 +480,7 @@ def __reversed__(self):
     __copy__ = copy
 
 
-# register the LRU cache as mutable mapping if possible
-try:
-    from collections import MutableMapping
-    MutableMapping.register(LRUCache)
-except ImportError:
-    pass
+abc.MutableMapping.register(LRUCache)
```

Comment:
I'm not sure why this `ImportError` handling was here, seeing that Python 2.6 always has `collections.MutableMapping` available. It was added a long time ago (>10 years) in 9bb7e4779182490abc6e1784b0ee63d22b91b11e, so I suppose that was needed for an older Python version but isn't anymore.